sites/main-site/src/content/blog/2025/nf-core-colab-guide.mdx

```diff
@@ -0,0 +1,202 @@
+headerImageAlt: "Epic Handshake between nf-core and colab"
+embedHeaderImage: true
+authors:
+    - "KurayiChawatama"
+label:
+    - "community post"
+---
+
+import { Image } from "astro:assets";
+import Profile from "@components/GitHubProfilePictureExtended.svelte";
+import colabmeme from "@assets/images/blog/nf-core-colab/colab-meme.jpg";
+import disconnectmeme from "@assets/images/blog/nf-core-colab/colab-disconnected-meme.png";
+import sleepmeme from "@assets/images/blog/nf-core-colab/dont-fall-asleep-meme.png";
+
+## Running nf-core pipelines in Google Colab
+
+Running and developing nf-core pipelines can be computationally intensive, requiring resources not easily available to students, newcomers, or participants in hands-on training workshops.
+Google Colab provides a free, accessible way to leverage powerful cloud hardware for computational tasks, making it an attractive option for students, researchers, and anyone with limited local resources.
+
+To make it easier for people to access such resources, we have just published a new detailed tutorial for running and developing nf-core/Nextflow pipelines in Google Colab is available at [this link](https://nf-co.re/docs/tutorials/google_colab/nf-core-colab-guide)!
+
+In this blog post, we provide background into our own experiences of running and developing Nextflow and nf-core pipelines, describing their pros and cons, but that also motivated the creation of the tutorial.
+
+### Why run pipelines in Google Colab
+
+<Image
+    src={colabmeme}
+    class="d-block m-auto"
+    width={600}
+    density={[1.5, 2]}
+    alt="Butch eats GPUs from the garbage while Tom is handfed in luxury"
+/>
+
+Google Colab provides free credits for basic computational infrastructure, making it an interesting option for people with limited resources to run computational workflows.
+Furthermore, for a fee, you can access the latest hardware each year at a fraction of the cost of buying a new PC.
+
+As bioinformatics and data science tasks become more resource-intensive, Colab offers a potentially cost-effective solution for learning and developing large-scale pipelines like those built with Nextflow.
+If you’re a student or developer working on a laptop, Colab can dramatically speed up your workflow.
+For example, you can write and test your code locally, then run it in Colab to take advantage of faster execution and larger memory—saving time and reducing frustration from crashes on limited hardware.
+
+Colab is also ideal for training workshops where participants may not have access to high-performance computing (HPC) clusters.
+Instructors can use real-world, large datasets instead of toy examples, thus giving everyone hands-on experience with industry-scale workflows.
+
+Ultimately, using Colab can help democratize access to advanced pipeline development and best practices, enabling more people to contribute to open-source projects like nf-core.
```

Comment:
More a question to the maintainers: Since nf-core's Code of Conduct specifically mentions socioeconomic status, it could be mentioned in this context?